### DISCUSSION TITLE
404

### DISCUSSION THREAD

**Discussion by Camell-Cloud** [Camell-Cloud/Camell-web3#12] [Q&A] [answered]
I think we need to add a 404 screen for our webpage?

**Comment by jeonilshin** [comment]
I think lotties are the best option

**Reply by Camell-Cloud** [reply]
could you give me a example?

**Reply by Camell-Cloud** [reply]
<img width="999" alt="image" src="https://github.com/user-attachments/assets/86c1bb4a-fa5d-4481-833a-35fbfd68a595" />

#### but with glitch effect

**Reply by jeonilshin** [reply, marked as answer]
done!

https://github.com/user-attachments/assets/9f7a7ade-01c0-4cb7-9cbf-1c86f1b72be9
